container-chains/templates/simple/runtime/src/lib.rs

```diff
@@ -408,6 +428,47 @@ impl pallet_cc_authorities_noting::Config for Runtime {
     type WeightInfo = ();
 }
 
+pub struct IdentityLookup;
```

Comment:
this, since its repeated in both templates, should also be move to a primitives section probably